src/rest/bucket.ts

```diff
@@ -9,14 +9,15 @@ import type { RESTManager } from './manager.ts'
 import { RequestQueue } from './queue.ts'
 import { APIRequest } from './request.ts'
 
-function parseResponse(res: Response, raw: boolean): any {
+// It returns JSON objects which are untyped so
+async function parseResponse(res: Response, raw: boolean): Promise<any> {
```

Comment:
just curious why use function not arrow function

Reply:
because yes, it doesn't really matter in this case tbh lol

Reply:
~~but it looks better among the other functions~~

Reply:
i did a quick bench for fun function definitions are 10ns faster than arrow functions (can't say for sure it applies to everything)

Reply:
hmmm kay